### ISSUE TITLE
where is lr_decay in the options_vot.yaml？

### ISSUE THREAD

**Issue by xi-mao** [HyeonseobNam/py-MDNet#39]
https://github.com/HyeonseobNam/py-MDNet/blob/28ff286ae0808f604690326684acb0a63ccf9b42/pretrain/train_mdnet.py#L40

where is the  parameter lr_decay? Ican't find it。

**Comment by hyeonseobnam** [comment]
Hi, @xi-mao 
It can be optionally given in options.yaml, which is not used by default.
For example, lr_decay: [30, 60] means that the learning rate is decayed at 30th and 60th cycles.
